spec/feature/chitter_spec.rb

```diff
@@ -5,15 +5,15 @@
         describe '.all' do
           it 'returns all messages' do
             connection = PG.connect(dbname:'chitter_test')
-            connection.exec("INSERT INTO peep (peep) VALUES ('The zombies are coming');")
-            connection.exec("INSERT INTO peep (peep) VALUES('Bring food');")
-            connection.exec("INSERT INTO peep (peep) VALUES('And Alcohol');")
+            connection.exec("INSERT INTO chitter_test (peep) VALUES ('The zombies are coming');")
+            connection.exec("INSERT INTO chitter_test (peep) VALUES('Bring food');")
+            connection.exec("INSERT INTO chitter_test (peep) VALUES('And Alcohol');")
       
-            chitter = Chitter.all
+            chitters = Chitter.all
       
-            expect(peep).to include("The zombies are coming")
-            expect(peep).to include("Bring food")
-            expect(peep).to include("And Alcohol")
+            expect(chitter_test).to include("The zombies are coming")
```

Comment:
What you mean to test is probably the output of `Chitter.all`.

'chitter_test' hasn't been defined - not in this Ruby program. Postgres (psql) knows about it, but postgres isn't responsible for parsing this line (you aren't using a `connection.exec("some SQL")`). So I expect this throws an 'undefined' error message.